Mathlib/Data/List/Basic.lean

```diff
@@ -732,7 +732,7 @@ composing a `List.map` with another `List.map`, fully applied.
 This is the reverse direction of `List.map_map`.
 -/
 theorem comp_map (h : β → γ) (g : α → β) (l : List α) : map (h ∘ g) l = map h (map g l) :=
-  (map_map _ _ _).symm
+  map_map.symm
```

Comment:
All those arguments should stay explicit in `map_map` since it's a rewriting lemma